TTS/utils/io.py

```diff
@@ -43,7 +33,15 @@ def load_config(config_path: str) -> AttrDict:
         with open(config_path, "r") as f:
             data = yaml.safe_load(f)
     else:
-        data = read_json_with_comments(config_path)
+        # fallback to json
+        with open(config_path, "r") as f:
+            input_str = f.read()
+        # handle comments
+        input_str = re.sub(r'\\\n', '', input_str)
+        input_str = re.sub(r'//.*\n', '\n', input_str)
+        input_str = input_str.replace('\\','/')
```

Comment:
Right here.

Reply:
right here what exactly?

Reply:
and why this PR? I don't see it is fixing anything? If there is pls fix the thing without removing the original code.

Reply:
input_str = input_str.replace('\\','/')
is resolving the backslash issue.

Reply:
now I see. Can you fit it without removing the function?

Reply:
What do you mean?